src/components/look-controls.js

```diff
@@ -20,6 +20,15 @@ module.exports.Component = registerComponent('look-controls', {
     this.setupMouseControls();
     this.setupHMDControls();
     this.bindMethods();
+
+    // Enable grab cursor class on canvas.
+    var sceneEl = this.el.sceneEl;
```

Comment:
@dmarcos 

This goes against the grain of the current style. See https://github.com/aframevr/aframe/issues/1679

I really like it this way because it consolidates everything related together in one visual chunk. If we wanted to extract, refactor, or remove this block, we simply take out this block and nothing else. If we moved `var sceneEl` above everything else, it'd take some reading that it's only used here.

I also put the function on top because it's a bit more logic. Declare it, and then use it.